cmd/filter/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	timodel "github.com/pingcap/tidb/parser/model"
+	"github.com/pingcap/tiflow/pkg/cmd/util"
+	"github.com/pingcap/tiflow/pkg/config"
+	"github.com/pingcap/tiflow/pkg/filter"
+	"github.com/spf13/cobra"
+)
+
+var (
+	table   string
+	ddl     string
+	target  string
+	cfgPath string
+)
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:   "TiCDC filter helper, use to check whether your filter config works as expected",
+		Short: "A tool to check table and ddl query against filter rules",
+		Run:   runFilter,
+	}
+
+	rootCmd.Flags().StringVarP(&target, "target", "T", "",
```

Comment:
The target can be obtained directly by determining whether table and ddl are empty, so it is an unnecessary parameter.